.gitignore

```diff
@@ -17,6 +17,7 @@
 /run/*.pid
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/log/purl_finder
```

Comment:
this probably should be `/log/purl_finder*` so to ignore all the purl finder output files.